### ISSUE TITLE
Missing component diagram with new instantiation-based OMEdit GUI

### ISSUE THREAD

**Issue by niklwors** [OpenModelica/OpenModelica#9531]
### Description
When inserting a component from the BRSL library onto a sheet in OMEdit, only the diagram of the Rexroth World is drawn but not the diagram of the component.  The Rexroth World is required as an outer element for each component. If you drag and drop the Rexroth World onto the sheet first, it will work.

### Steps to Reproduce

Load the BRSL package: Rexroth_Generic/package.mo.
Drag and Drop a component e.g see the attached picture 1.

### Expected Behavior

Both components should be on the sheet.

### Screenshots
**Picture 1**
<img width="1208" alt="world_ticket-1" src="https://user-images.githubusercontent.com/12269776/196213229-a04df940-328f-436d-815d-59870c3ef294.png">
**Picture 2**
<img width="1146" alt="wolrd_ticket-2" src="https://user-images.githubusercontent.com/12269776/196213389-b2cb0e4a-cf50-4f9a-96ba-088608f22d9b.png">

**Picture 3**
<img width="1166" alt="world_ticket-3" src="https://user-images.githubusercontent.com/12269776/196213640-10f94359-c1ba-40ae-a660-e9b2793a946d.png">
Version and OS

 - OpenModelica Version: v1.20.0-dev-348-g7cc2ddob5b (64-bit)  
 - OS: Windows 10, 64 bit
 - MSL 4.0.0
 - BRSL 2.2 dev

### Additional Context

@sbaetzing @abuntrock

**Comment by adeas31** [comment]
This one is very weird. @perost can you please debug this,

```
model M
  Rexroth_Generic.Mechanics.Rotational.Components.VariableFrictionRotational variableFrictionRotational annotation(
    Placement(visible = true, transformation(origin = {62, 24}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
annotation(
    uses(Rexroth_Generic(version = "2.1.0")));
end M;
```

`getModelInstance(M)` returns the 2 components i.e., `variableFrictionRotational` and `BRSL_World`. This is wrong and makes OMEdit and the generated Modelica code out of sync. Perhaps the compiler needs to use the default `BRSL_World` for instantiation but this information shouldn't be dumped in the JSON.

**Comment by perost** [comment]
> This one is very weird. @perost can you please debug this,
> 
> ```
> model M
>   Rexroth_Generic.Mechanics.Rotational.Components.VariableFrictionRotational variableFrictionRotational annotation(
>     Placement(visible = true, transformation(origin = {62, 24}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
> annotation(
>     uses(Rexroth_Generic(version = "2.1.0")));
> end M;
> ```
> 
> `getModelInstance(M)` returns the 2 components i.e., `variableFrictionRotational` and `BRSL_World`. This is wrong and makes OMEdit and the generated Modelica code out of sync. Perhaps the compiler needs to use the default `BRSL_World` for instantiation but this information shouldn't be dumped in the JSON.

Fixed in #9599, now we just skip dumping generated inner components. That might cause other issues since the model instance now contain references to things that no longer exist in the JSON, so it could be that we still need to dump it but maybe avoid drawing it instead. But it's trivial to toggle it on or off, so I guess we'll see.

**Comment by adeas31** [comment]
Tested this again and it works now as expected.
